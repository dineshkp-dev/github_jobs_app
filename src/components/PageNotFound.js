import React from 'react';
import { Link } from 'react-router-dom';
import '../styles/pagenotfound.css';

function PageNotFound() {
  return (
    <div className='pagenotfound-container'>
      <h1 className='pagenotfound-title'>The page does not exist</h1>
      <div className='pagenotfound-msg'>
        Please click <Link to='/'>here</Link> to go to the home page
      </div>
    </div>
  );
}

export default PageNotFound;
