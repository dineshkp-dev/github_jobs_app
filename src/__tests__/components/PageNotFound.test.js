import PageNotFound from '../../components/PageNotFound';
import { BrowserRouter as Router } from 'react-router-dom';
import { render } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';

describe('PageNotFound snapshot test', () => {
  test('renders', () => {
    const { container } = render(
      <Router>
        <PageNotFound />
      </Router>
    );
    expect(container).toMatchSnapshot();
  });
});
