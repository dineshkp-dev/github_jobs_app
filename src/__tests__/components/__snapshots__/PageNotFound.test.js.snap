// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageNotFound snapshot test renders 1`] = `
<div>
  <div
    class="pagenotfound-container"
  >
    <h1
      class="pagenotfound-title"
    >
      The page does not exist
    </h1>
    <div
      class="pagenotfound-msg"
    >
      Please click 
      <a
        href="/"
      >
        here
      </a>
       to go to the home page
    </div>
  </div>
</div>
`;
